test/api/settings_test.rb

```diff
@@ -0,0 +1,21 @@
+require 'test_helper'
+
+class SettingsTest < ActiveSupport::TestCase
+  include Rack::Test::Methods
+  include TestHelpers::AuthHelper
+  include TestHelpers::JsonHelper
+
+  def app
+    Rails.application
+  end
+
+  def test_get_privacy_policy_details
+    get 'api/settings/privacy'
+    assert_equal 200, last_response.status
```

Comment:
This tests for 200, but should also test that we are getting a privacy statement back

Reply:
More checks have been added in commit #26b873a below